modBoot/src/org/aion/Aion.java

```diff
@@ -71,34 +69,34 @@ public static void main(String args[]) throws InterruptedException {
          * if in the config.xml id is set as default [NODE-ID-PLACEHOLDER]
          * return true which means should save back to xml config
          */
-        if(cfg.fromXML())
-            cfg.toXML(new String[]{ "--id=" + cfg.getId() });
+        if (cfg.fromXML()) cfg.toXML(new String[] {"--id=" + cfg.getId()});
 
-        
         try {
             ServiceLoader.load(AionLoggerFactory.class);
         } catch (Exception e) {
             System.out.println("load AionLoggerFactory service fail!" + e.toString());
             throw e;
         }
 
+        // If commit this out, the config setting will be ignore. all log module been set to "INFO"
+        // Level
 
-        // If commit this out, the config setting will be ignore. all log module been set to "INFO" Level
-        AionLoggerFactory.init(cfg.getLog().getModules());
+        /** Change INITIALIZE signature to include LOGFILE */
+        AionLoggerFactory.init(cfg.getLog().getModules(), cfg.getLog().getLogFile(), cfg.getLog().getLogPath());
```

Comment:
should check the logParh is a valid path

https://stackoverflow.com/questions/1976007/what-characters-are-forbidden-in-windows-and-linux-directory-names

Reply:
It now removes the special characters in logPath